### PR TITLE
Master

### PR DESCRIPTION
**Pull request by Pbmangan**
### New Submission Check List

Please ensure your submission ticks all of these boxes:

 - [ ] This is Software as a Service not self hosted
 - [ ] It has a free tier not just a free trial
 - [ ] The submission mentions what is free
 - [ ] The submission is not already present in the list
 - [ ] The service has contact details of those running it and a privacy policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1192,6 +1193,7 @@ Table of Contents
    * [kanbanflow.com](https://kanbanflow.com/) — Board-based project management. Free, premium version with more options
    * [kanbantool.com](https://kanbantool.com/) — Kanban board-based project management. Free, paid plans with more options
    * [Kitemaker.co](https://kitemaker.co) - Collaborate through all phases of the product development process and keep track of work across Slack, Discord, Figma, and Github. Unlimited users, unlimited spaces. Free plan up to 250 work items.
+  * [Kiter.app](https://www.kiter.app/) - Let's anyone organize their job search and keep track of interviews, opportunities, and connections. Powerful web app and chrome extension. Completely free.
```
Comment (ripienaar) [inline README.md:1196]: thanks, please fix the indent here

**Comment by Pbmangan** [comment]
Added:

PromptLoop.com (I am a founder YC start with generous free tier)
Kiter.app (I was one of the founders - free job seeker tool)

File: README.md (hunk 1 above)
Comment (Pbmangan) [reply README.md]: Resolved

**Comment by ripienaar** [comment]
thanks

**Pull request merged**